src/app/pages/server-compare/server-compare.component.ts

```diff
@@ -143,6 +146,13 @@ export class ServerCompareComponent implements OnInit {
     return `${(item.storage_size / 1000).toFixed(1)} TB`;
   }
 
+  bytesToSize(bytes: number) {
+    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
```

Comment:
when the base is 1024 (instead of 1000), it's `kiB`, `MiB` etc

```suggestion
    const sizes = ['Bytes', 'kiB', 'MiB', 'GiB', 'TiB'];
```

Reply:
might be useful to have `bytesToSize` as a pipe?